blt/src/Blt/Plugin/Commands/CircleCiCommands.php

```diff
@@ -377,8 +377,7 @@ protected function installDrupal($profile = 'minimal') {
       ->drush('site-install')
       ->args($profile)
       ->option('verbose')
-      ->option('yes')
-      ->option('db-url', static::DB_URL, '=');
```

Comment:
Is the DB URL option now handled by something else?

Reply:
through `docroot/sites/settings/ci.settings.php`